### PR TITLE
Add Android setup & Manual adjustments section

### PR DESCRIPTION
**Pull request by trmi1012**
* Add Android setup and manual adjustments section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,52 @@
...
+
+Run
+
+`npm run build`
+
+Sync the plugin and open it in Android Studio:
+
+`npx cap sync`
+
+`npx cap open android`
```
Comment (EmDee) [inline README.md]: This section is not needed, right? It's the same as the `# Local Installation`

Comment (trmi1012) [reply README.md]: I dont see `# Local Installation` section in the README.md

Comment (EmDee) [reply README.md]: Ah, it's in the Resume within the vue-starter directory.

I think we should replace this readme with the one in there.

Comment (trmi1012) [reply README.md]: updated

**Pull request merged**